test/TestProjects/CollapseRequestCondition-LowLevel/Generated/RequestConditionCollapseClient.cs

```diff
@@ -128,6 +128,206 @@ public virtual Response CollapseGet(RequestConditions requestConditions = null,
             }
         }
 
+        /// <param name="requestConditions"> The content to send as the request conditions of the request. </param>
+        /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+#pragma warning disable AZC0002
+        public virtual async Task<Response> MissIfNonMatchGetAsync(RequestConditions requestConditions = null, RequestContext context = null)
+#pragma warning restore AZC0002
+        {
+            using var scope = _clientDiagnostics.CreateScope("RequestConditionCollapseClient.MissIfNonMatchGet");
```

Comment:
Shouldn't it generate `_clientDiagnostics` -> `ClientDiagnostics` ?

Reply:
resolved